files/default/supervisor-ckan-harvest.conf

```diff
@@ -53,7 +54,9 @@ startsecs=10
 [program:ckan-harvest-gather]
 
 ; Use the full paths to the virtualenv and your configuration file here.
-command=/usr/lib/ckan/default/bin/paster --plugin=ckanext-harvest harvester gather_consumer --config=/etc/ckan/default/production.ini
+command=/usr/lib/ckan/default/bin/ckan_cli harvester gather_consumer
+
+environment=PASTER_PLUGIN=ckanext-harvest
```

Comment:
should this be environment="PAS...."  as it has another = in the middle?

Reply:
The current syntax matches the example at http://supervisord.org/configuration.html#program-x-section-example